src/pages/PrescriptionPage.js

```diff
@@ -4,213 +4,189 @@
+    () =>
+      ALPHABET.map(character => (
+        <TableShortcut key={character} onPress={alphabetCallback}>
+          {character.toUpperCase()}
+        </TableShortcut>
+      )),
+    []
   );
 
-  const { verticalContainer, topButton } = globalStyles;
-  const {
-    pageTopSectionContainer,
-    pageTopLeftSectionContainer,
-    pageTopRightSectionContainer,
-    searchBar,
-  } = globalStyles;
   return (
-    <DataTablePageView>
-      <View style={pageTopSectionContainer}>
-        <View style={pageTopLeftSectionContainer}>
-          <PageInfo columns={pageInfoColumns} isEditingDisabled={isFinalised} />
-          <SearchBar onChangeText={onFilterData} style={searchBar} value={searchTerm} />
-        </View>
-        <View style={pageTopRightSectionContainer}>
-          <View style={verticalContainer}>
-            <PageButton
-              style={{ ...topButton, marginLeft: 0 }}
-              text={buttonStrings.new_item}
-              onPress={onSelectNewItem}
-              isDisabled={isFinalised}
-            />
-          </View>
+    <View style={{ ...row, marginTop: 50 }}>
+      <TableShortcuts>
+        <TableShortcut>
+          <FavouriteStarIcon />
+        </TableShortcut>
+        <AlphabetShortcuts />
+      </TableShortcuts>
+      <View style={mediumFlex}>
+        <SimpleTable
+          data={UIDatabase.objects('Item')}
+          columns={columns}
+          selectRow={x => chooseItem(x)}
+          ref={tableRef}
+        />
+      </View>
+      <View style={largeFlex}>
+        <View>
+          {transaction.items.map(item => (
+            <View>
+              <Text>{item.itemName}</Text>
+            </View>
+          ))}
         </View>
       </View>
-      <DataTable
-        data={data}
-        extraData={dataState}
-        renderRow={renderRow}
-        renderHeader={renderHeader}
-        keyExtractor={keyExtractor}
-        getItemLayout={getItemLayout}
-        columns={columns}
-      />
-      <BottomConfirmModal
-        isOpen={hasSelection}
-        questionText={modalStrings.remove_these_items}
-        onCancel={onDeselectAll}
-        onConfirm={onDeleteItems}
-        confirmText={modalStrings.remove}
-      />
-      <DataTablePageModal
-        fullScreen={false}
-        isOpen={!!modalKey}
-        modalKey={modalKey}
-        onClose={onCloseModal}
-        onSelect={getModalOnSelect()}
-        dispatch={dispatch}
-        currentValue={modalValue}
-      />
-    </DataTablePageView>
+      <PageButton text="NEXT" onPress={() => changeTab(1)} />
+    </View>
   );
-};
-
-const mapDispatchToProps = (dispatch, ownProps) =>
-  getPageDispatchers(dispatch, ownProps, 'Transaction', ROUTES.PRESCRIPTION);
-
-const mapStateToProps = state => {
-  const { pages } = state;
-  const { prescription } = pages;
+});
 
-  return prescription;
+const Summary = connect(mapStateToProps)(({ transaction }) => (
+  <View style={{ flex: 1 }}>
+    {transaction.items.map(item => (
+      <View>
```

Comment:
doesn't this need a `key` prop?

Reply:
ae8bc54918ac9b217f095692f2dba48f7c6e1739